eth/protocols/eth/protocol.go

```diff
@@ -239,19 +239,22 @@ type BlockBodiesRLPPacket66 struct {
 type BlockBody struct {
 	Transactions []*types.Transaction // Transactions contained within a block
 	Uncles       []*types.Header      // Uncles contained within a block
+	Withdrawals  []*types.Withdrawal  `rlp:"optional"` // Withdrawals contained within a block
```

Comment:
But... you cannot just redefine the blockbody like this. 
You need to define a new packet, e.g. 
`BlockBodiesPacket67`, containing

```
type BlockBodiesPacket67 struct {
	RequestId uint64
	[]*BlockBody67
}
type BlockBody67 struct {
     Transactions ...
     ...
     Withdrawals
}
```
Then you need to add a new protocol version, etc etc

Reply:
Ah.. So it's RLP optional.. Yeah ok this might actually just work out of the box then.. ?

Reply:
Yes it works out of the box I think, otherwise the testnets wouldn't work